specs/language/expressions.tex

```diff
@@ -160,3 +160,87 @@
 overloaded implementation of \texttt{operator[]} (\ref{Overload}).\footnote{HLSL
 does not support the base address of a subscript operator being the expression
 inside the braces, which is valid in C and C++.}
+
+\Sec{Function Calls}{Expr.Post.Call}
+
+\p A function call may be an \textit{ordinary function}, or a \textit{member
+function}. In a function call to an \textit{ordinary function}, the
+\textit{postfix-expression} must be an lvalue that refers to a function. In a
+function call to a \textit{member function}, the \textit{postfix-expression}
+will be an implicit or explicit class member access whose \textit{id-expression}
+is a member function name.
+
+\p When a function is called, each parameter shall be initialized with its
+corresponding argument. The order in which parameters are initialized is
+undefined. \footnote{Today in DXC targeting DXIL matches the Microsoft C++ ABI
```

Comment:
```suggestion
unspecified. \footnote{Today in DXC targeting DXIL matches the Microsoft C++ ABI
```

Consider the term "unspecified" - given the [C++ spec definition](https://timsong-cpp.github.io/cppwp/defns.unspecified#def:behavior,unspecified) would apply to such situations and is introduced/inherited in `\Sub{Unspecified Behavior}{Intro.Defs.Unspecified}` (introduction.tex).